### ISSUE TITLE
Chatbot for veterans resources

### ISSUE THREAD

**Issue by christyleos** [TechForJustice/project-ideas#28]
A chatbot that can make it easy to access and understand veterans resources, such as ones found on va.gov and other support websites.  This could be done via facebook, or for the website itself, and would answer questions that don't necessarily need human help, but guidance with a human feel to it. 

Perhaps we need to narrow down the scope - addressing all the issues a veterans can face is a bit much for a hackathon. 

The [chatbot design playbook](https://crew.co/blog/the-chatbot-design-playbook) might be helpful.

**Comment by MigrantCloth** [comment]
Hi.

Talk about perfect timing! I've been putting together an ebook to help vets better prefer for/manage TAPS (I'm a USMC vet). The intent was to use that and a landing page and funnel the audience to a website built to create a platform to aggregate resources I am a tech noob as I just started learning last year but (1) I'm a project coordinator and business analyst and (2) I intend to further build my tech skills to help veterans. 

Webframe idea:
[EAS Countdown.pptx](https://github.com/TechForJustice/project-ideas/files/802408/EAS.Countdown.pptx)

Intent: Veteran can enter their EAS date, the clock counts down. The buttons below the countdown will redirect to resources such as organizations for claims (VFW, AMVETS,etc), education steps, finance steps, and so on. 

When I first looked to address this issue, I saw individual issues I've helped veterans with or issues I had and there really is a broad range. I settled on starting on guidance to better manage TAPS(including teaching Cornell note-taking and thorough career search using an MOS engine and a job engine) as a way to both keep my scope small and get feedback. Then I I figured I could build 'backward' to creating a platform for veterans to better manage their entire careers. I think too broad a scope up front not only 'bites off more than you can chew' but also would give the audience too much at once to take in (and most already feel that way about the TAPS experience). 

What are you thoughts?

**Comment by JeffL-T** [comment]
Hello MigrantCloth, 

An EAS Countdown is a great idea to help relieve the volume of tasks facing members at separation.  Project idea 27, Veteran transition platform/app, is listed prior to this Chatbot idea and would be a good place to discuss your countdown idea.  Do you have any objection to me moving it there?  Also there, you will see a link to a transition tool being developed by Bunker Labs called VictorApp.  It looks like a good app, but I can't see that it has a countdown feature like you have conceived.  Such a feature would have tremendously aided my own transition so I think that it is a superb idea.

**Comment by MigrantCloth** [comment]
Hey Jeff,

No, that's fine. Sounds great!

I did get a chance to check out the website for VictorApp. I'm not able to see the app (Android) but from everything on the site, it looks good. Even if we're different sides of the same coin (or different sections of the same side) I think the countdown and interactive goal-setting journey through resources idea would be a great asset to any transition platform. This is what I'm hoping to achieve with EAS Countdown.

**Comment by Ryan-Koch** [comment]
I really like the idea of attempting to unify information sources through the use of a bot. This is a key area of investment in the private sector in large part due to the ease of use and understandability factors you mentioned. There is a swath of resources available for development, including a couple below:

https://dev.botframework.com/ <- Cross platform framework from Microsoft
https://developers.facebook.com/docs/messenger-platform <- Facebook messanger platform

**Comment by JeffL-T** [comment]
The BIG IDEA is that online, mobile, and AI technology resources can help veterans access resources and secure the benefits to which they are entitled to make a successful transition from military to civilian life.

**Comment by Ryan-Koch** [comment]
Agreed! If you folks are going to pursue this idea for the hack-a-thon I'd most assuredly be game to help out.
